Sources/MapboxNavigation/VisualInstruction.swift

```diff
@@ -85,10 +85,11 @@ extension VisualInstruction {
     /// Returns whether the `VisualInstruction`’s maneuver image should be flipped according to the driving side.
     public func shouldFlipImage(side: DrivingSide) -> Bool {
```

Comment:
Apparently this method is used by only CarPlay and user notifications, and it’s redundant to the flipping logic baked into `ManeuverView.draw(_:)`. That logic affects the view’s transform, which I guess isn’t reflected in the image exported by `UIImage.imageRepresentation`. But `UIImage.imageRepresentation` could apply the view’s transform, and moreover we could factor out the flipping logic if necessary.